stdlib/public/Concurrency/ContinuousClock.swift

```diff
@@ -0,0 +1,138 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Swift
+
+@available(SwiftStdlib 9999, *)
+public struct ContinuousClock {
+  public struct Instant: Codable, Sendable {
+    internal var _value: Swift.Duration
+
+    internal init(_value: Swift.Duration) {
+      self._value = _value
+    }
+  }
+
+  public init() { }
+}
+
+@available(SwiftStdlib 9999, *)
+extension Clock where Self == ContinuousClock {
+  @available(SwiftStdlib 9999, *)
+  public static var continuous: ContinuousClock { return ContinuousClock() }
+}
+
+@available(SwiftStdlib 9999, *)
+extension ContinuousClock: Clock {
+  public var now: ContinuousClock.Instant {
+    ContinuousClock.now
+  }
+
+  public var minimumResolution: Swift.Duration {
+    var seconds = Int64(0)
+    var nanoseconds = Int64(0)
+    _getClockRes(
+      seconds: &seconds,
+      nanoseconds: &nanoseconds,
+      clock: .continuous)
+    return .seconds(seconds) + .nanoseconds(nanoseconds)
+  }
+
+  public static var now: ContinuousClock.Instant {
+    var seconds = Int64(0)
+    var nanoseconds = Int64(0)
+    _getClockRes(
```

Comment:
IIUC, this probably should be calling `_getTime` instead of `_getClockRes`.

Reply:
oops! that is definitely a bug. Good catch - fixed.